### PR TITLE
Add Beekeeper Studio icon

### LINKED ISSUE #8061: Request: Beekeeper Studio icon
### Brand Name

Beekeeper Studio

### Website

https://beekeeperstudio.io

### Popularity Metric

Beekeeper Studio has 11.9K GitHub stars
https://github.com/beekeeper-studio/beekeeper-studio

### Official Resources for Icon and Color

svg: https://www.beekeeperstudio.io/static/press-kit/bk-logo-icon-lightbg.svg

Beekeeper Yellow: `#fad83b`, from Beekeeper Studio press kit page: https://www.beekeeperstudio.io/press/

### Additional Comments

There are a few Beekeeper-approved options for the logo color per their press page, but the most recognizable, on-brand logo color is `#fad83b` on a black background, like a bumblebee.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by jrhaberland**
beekeeperstudio

**Issue:** closes #8061

**Popularity metric:**
Beekeeper Studio has 15.6K GitHub stars
https://github.com/beekeeper-studio/beekeeper-studio

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2053,6 +2053,14 @@
             "source": "https://beatstars.world/brand-guidelines",
             "guidelines": "https://beatstars.world/brand-guidelines"
         },
+        {
+            "title": "Beekeeper Studio",
+            "hex": "FAD83B",
+            "source": "https://www.beekeeperstudio.io",
```
Comment (uncenter) [inline _data/simple-icons.json:2059]: We should use their [press kit](https://www.beekeeperstudio.io/press/) as source here.

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:1]: Looks like their [trademark guidelines](https://www.beekeeperstudio.io/legal/trademark/) do allow us to display it this way (I would like a second read through though of those guidelines if you could @adamrusted), however we should link the page under `guidelines`.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by jrhaberland** [comment]
Need to fix the color codes again I updated it to what it was on the brand guidlines page, then realized the colors don't line up with the hex codes ☹️ 
image

**Comment by uncenter** [comment]
> Need to fix the color codes again I updated it to what it was on the brand guidlines page, then realized the colors don't line up with the hex codes ☹️ image

Ahah yeah I noticed that earlier and opened https://github.com/beekeeper-studio/beekeeper-studio/issues/2268!

File: _data/simple-icons.json
Comment (adamrusted) [reply _data/simple-icons.json:1]: Yep, happy with the reading on those guidelines - but would suggest we add a `guidelines` entry - as @uncenter says above.

Comment (jrhaberland) [reply _data/simple-icons.json:1]: I added a guidelines entry, so should be all good to go

**Review by uncenter — CHANGES_REQUESTED** [review]
SVG looks good, just this one little change!

**Review by uncenter — APPROVED** [review]

**Pull request merged**